Ajout du JSON schema pour les thèses

ggrossetie opened the pull request.

ggrossetie committed:
chore: ajout des métadonnées pour les thèses

gromettoclara commented on front/src/schemas/corpus-thesis-ui-schema.json:
```diff
@@ -0,0 +1,239 @@
+  "@version": {
+    "ui:widget": "hidden"
+  },
+  "discipline": {
```
comme discuté ce matin : ce serait top d'avoir un type array

ggrossetie replied:
dans ce cas je renomme en `disciplines` pour que ça soit explicite

gromettoclara commented on front/src/schemas/corpus-thesis-ui-schema.json:
```diff
@@ -0,0 +1,239 @@
+      }
+    }
+  },
+  "affiliatedInstitutions": {
```
pareil : comme on a dit ce matin ce serait super d'avoir un type array, quitte à changer la logique par rapports aux métadonnées de l'article

ggrossetie replied:
c'est déjà le cas, je pense que tu voulais faire référence à `affiliations`

ggrossetie committed:
chore: déplace les traductions du formulaires des métadonnées

- simplifie le nommage
- partage de traduction entre les formulaires
- un fichier dedié pour les formulaires (form.json)
